packages/firebase_analytics/lib/firebase_analytics.dart

```diff
@@ -92,7 +92,7 @@ class FirebaseAnalytics {
   /// https://firebase.google.com/docs/reference/android/com/google/firebase/analytics/FirebaseAnalytics.html#setCurrentScreen(android.app.Activity, java.lang.String, java.lang.String)
   /// https://firebase.google.com/docs/reference/ios/firebaseanalytics/api/reference/Classes/FIRAnalytics#setscreennamescreenclass
   Future<Null> setCurrentScreen(
-      {@required String screenName, String screenClassOverride}) async {
+      {@required String screenName, String screenClassOverride: 'Flutter'}) async {
```

Comment:
For the flutter/plugins repo we're using the dart formatter (enforced by Travis), which doesn't put the spaces there.